source/layouts/_footer.erb

```diff
@@ -0,0 +1,31 @@
+<div class="app-c-footer__navigation">
+
+  	<div class="app-c-footer__navigation--column">
+	  	<h2 class="heading-medium">GOV.UK Design System</h2>
+	  	<ul>
+	  		<li><a href="/design-patterns/patterns/sitemap.html">Site contents</a></li>
```

Comment:
Given the sitemap doesn't exist yet, and to avoid introducing commented out code, can we remove this whole block of links for now?